src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -53,4 +55,145 @@ public String getColor() {
     public List<Section> getSections() {
```

Comment:
```suggestion
    public Sections getSections() {
```

일급콜렉션을 만들어보면 어떨까요? 
그리고, 그에 대한 단위 테스트도 작성하면 좋겠네요 :)